<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" connectslotsbyname="false">
 <author>hgjazhgj</author>
 <class>fgoMainWindow</class>
 <widget class="QMainWindow" name="fgoMainWindow">
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>FGO-py - hgjazhgj</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{font-family:&quot;Microsoft YaHei UI Light&quot;;font-size:15px}</string>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="LAYOUT_INFO">
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="LBL_APPLE">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>苹果</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="LAYOUT_INFO_APPLE">
        <item>
         <widget class="QComboBox" name="CBX_APPLE">
          <property name="statusTip">
           <string>要吃的苹果种类,在运行前设置</string>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>金</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>银</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>铜</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>彩</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="TXT_APPLE">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::NoContextMenu</enum>
          </property>
          <property name="statusTip">
           <string>要吃的苹果数量,在运行前设置</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="maximum">
           <number>114514</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="LBL_TEAM">
        <property name="text">
         <string>编队位置</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="TXT_TEAM">
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="statusTip">
         <string>所选编队在队伍编成界面的位置,从左到右1-10,0为无效</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="LBL_CURRENTDEVICE">
        <property name="text">
         <string>当前设备</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="LBL_DEVICE">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BTN_CONNECT">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="statusTip">
           <string>连接到设备</string>
          </property>
          <property name="text">
           <string>更改</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="LAYOUT_FUNC">
      <item>
       <layout class="QVBoxLayout" name="LAYOUT_FUNC_BATTLE">
        <item>
         <widget class="QPushButton" name="BTN_CLASSIC">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="statusTip">
           <string>经典战斗</string>
          </property>
          <property name="text">
           <string>经典战斗</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BTN_MAIN">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="statusTip">
           <string>在关卡列表界面将要刷的关卡置于第一个来清空体力</string>
          </property>
          <property name="text">
           <string>肝!</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="LAYOUT_FUMC_MAIN">
        <item>
         <widget class="QPushButton" name="BTN_PAUSE">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="statusTip">
           <string>暂停/继续战斗</string>
          </property>
          <property name="text">
           <string>挂起战斗</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BTN_STOP">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="statusTip">
           <string>立刻终止战斗</string>
          </property>
          <property name="text">
           <string>终止战斗</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="LAYOUT_FUNC_CONTROL">
        <item>
         <widget class="QPushButton" name="BTN_SCREENSHOT">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="statusTip">
           <string>检查截图确定连接建立</string>
          </property>
          <property name="text">
           <string>检查截图</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BTN_STOPLATER">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="statusTip">
           <string>在完成若干场战斗后终止战斗</string>
          </property>
          <property name="text">
           <string>预约终止</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MENU">
   <widget class="QMenu" name="MENU_ABOUT">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="MENU_ABOUT_ABOUT"/>
    <addaction name="MENU_ABOUT_LICENSE"/>
   </widget>
   <widget class="QMenu" name="MENU_FILE">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="MENU_FILE_EXPLORER"/>
   </widget>
   <widget class="QMenu" name="MENU_SCRIPT">
    <property name="title">
     <string>程序</string>
    </property>
    <addaction name="MENU_SCRIPT_GACHA"/>
    <addaction name="MENU_SCRIPT_LOTTERY"/>
    <addaction name="MENU_SCRIPT_MAILFILTER"/>
    <addaction name="MENU_SCRIPT_SYNTHESIS"/>
    <addaction name="MENU_SCRIPT_GACHAHISTORY"/>
    <addaction name="MENU_SCRIPT_EXPBALL"/>
   </widget>
   <widget class="QMenu" name="MENU_CONTROL">
    <property name="title">
     <string>控制</string>
    </property>
    <widget class="QMenu" name="MENU_CONTROL_169">
     <property name="title">
      <string>全面屏适配</string>
     </property>
     <addaction name="MENU_CONTROL_169_INVOKE"/>
     <addaction name="MENU_CONTROL_169_REVOKE"/>
    </widget>
    <addaction name="MENU_CONTROL_STAYONTOP"/>
    <addaction name="MENU_CONTROL_TRAY"/>
    <addaction name="separator"/>
    <addaction name="MENU_CONTROL_MAPKEY"/>
    <addaction name="MENU_CONTROL_169"/>
    <addaction name="MENU_CONTROL_NOTIFY"/>
    <addaction name="separator"/>
    <addaction name="MENU_CONTROL_BENCH"/>
    <addaction name="MENU_CONTROL_EXEC"/>
   </widget>
   <widget class="QMenu" name="MENU_SETTINGS">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="MENU_SETTINGS_DEFEATED"/>
    <addaction name="MENU_SETTINGS_KIZUNAREISOU"/>
    <addaction name="MENU_SETTINGS_SPECIALDROP"/>
   </widget>
   <addaction name="MENU_FILE"/>
   <addaction name="MENU_SCRIPT"/>
   <addaction name="MENU_SETTINGS"/>
   <addaction name="MENU_CONTROL"/>
   <addaction name="MENU_ABOUT"/>
  </widget>
  <widget class="QStatusBar" name="STATUS"/>
  <action name="MENU_ABOUT_ABOUT">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="MENU_FILE_EXPLORER">
   <property name="text">
    <string>资源管理器</string>
   </property>
  </action>
  <action name="MENU_CONTROL_STAYONTOP">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>窗口置顶</string>
   </property>
  </action>
  <action name="MENU_SCRIPT_GACHA">
   <property name="text">
    <string>抽友情</string>
   </property>
   <property name="statusTip">
    <string>先抽一发友情十连,在结算界面运行本功能</string>
   </property>
  </action>
  <action name="MENU_SCRIPT_SYNTHESIS">
   <property name="text">
    <string>礼装强化</string>
   </property>
   <property name="statusTip">
    <string>在选择了强化对象未选择强化材料的界面运行本功能</string>
   </property>
  </action>
  <action name="MENU_SCRIPT_GACHAHISTORY">
   <property name="text">
    <string>召唤记录</string>
   </property>
   <property name="statusTip">
    <string>统计导出召唤记录,在抽卡记录页面运行</string>
   </property>
  </action>
  <action name="MENU_SCRIPT_EXPBALL">
   <property name="text">
    <string>搓丸子</string>
   </property>
   <property name="statusTip">
    <string>把若干张低星礼装合并成一个</string>
   </property>
  </action>
  <action name="MENU_CONTROL_MAPKEY">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>加载按键映射</string>
   </property>
  </action>
  <action name="MENU_SCRIPT_LOTTERY">
   <property name="text">
    <string>抽奖池</string>
   </property>
  </action>
  <action name="MENU_SCRIPT_MAILFILTER">
   <property name="text">
    <string>领狗粮</string>
   </property>
  </action>
  <action name="MENU_CONTROL_BENCH">
   <property name="text">
    <string>Bench</string>
   </property>
  </action>
  <action name="MENU_CONTROL_EXEC">
   <property name="text">
    <string>Execute</string>
   </property>
  </action>
  <action name="MENU_SETTINGS_DEFEATED">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>战败撤退时终止战斗</string>
   </property>
  </action>
  <action name="MENU_SETTINGS_KIZUNAREISOU">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>获得羁绊礼装时终止战斗</string>
   </property>
  </action>
  <action name="MENU_SETTINGS_SPECIALDROP">
   <property name="text">
    <string>若干特殊掉落后终止战斗</string>
   </property>
  </action>
  <action name="MENU_ABOUT_LICENSE">
   <property name="text">
    <string>使用许可</string>
   </property>
  </action>
  <action name="MENU_FILE_QUIT">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="MENU_CONTROL_TRAY">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>关闭到托盘</string>
   </property>
  </action>
  <action name="MENU_CONTROL_169_INVOKE">
   <property name="text">
    <string>调整为16:9</string>
   </property>
  </action>
  <action name="MENU_CONTROL_169_REVOKE">
   <property name="text">
    <string>恢复原分辨率</string>
   </property>
  </action>
  <action name="MENU_CONTROL_NOTIFY">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>消息推送</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>TXT_TEAM</tabstop>
  <tabstop>CBX_APPLE</tabstop>
  <tabstop>TXT_APPLE</tabstop>
  <tabstop>BTN_CLASSIC</tabstop>
  <tabstop>BTN_MAIN</tabstop>
  <tabstop>BTN_PAUSE</tabstop>
  <tabstop>BTN_STOP</tabstop>
  <tabstop>BTN_SCREENSHOT</tabstop>
  <tabstop>BTN_STOPLATER</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>BTN_CLASSIC</sender>
   <signal>clicked()</signal>
   <receiver>fgoMainWindow</receiver>
   <slot>runClassic()</slot>
  </connection>
  <connection>
   <sender>BTN_MAIN</sender>
   <signal>clicked()</signal>
   <receiver>fgoMainWindow</receiver>
   <slot>runMain()</slot>
  </connection>
  <connection>
   <sender>BTN_SCREENSHOT</sender>
   <signal>clicked()</signal>
   <receiver>fgoMainWindow</receiver>
   <slot>screenshot()</slot>
  </connection>
  <connection>
   <sender>BTN_PAUSE</sender>
   <signal>clicked(bool)</signal>
   <receiver>fgoMainWindow</receiver>
   <slot>pause(bool)</slot>
  </connection>
  <connection>
   <sender>BTN_STOP</sender>
   <signal>clicked()</signal>
   <receiver>fgoMainWindow</receiver>
   <slot>stop()</slot>
  </connection>
  <connection>
   <sender>BTN_CONNECT</sender>
   <signal>clicked()</signal>
   <receiver>fgoMainWindow</receiver>
   <slot>connect()</slot>
  </connection>
  <connection>
   <sender>MENU_FILE_EXPLORER</sender>
   <signal>triggered()</signal>
   <receiver>fgoMainWindow</receiver>
   <slot>explorerHere()</slot>
  </connection>
  <connection>
   <sender>MENU_ABOUT_ABOUT</sender>
   <signal>triggered()</signal>
   <receiver>fgoMainWindow</receiver>
   <slot>about()</slot>
  </connection>
  <connection>
   <sender>MENU_SCRIPT_GACHA</sender>
   <signal>triggered()</signal>
   <receiver>fgoMainWindow</receiver>
   <slot>runGacha()</slot>
  </connection>
  <connection>
   <sender>MENU_SCRIPT_SYNTHESIS</sender>
   <signal>triggered()</signal>
   <receiver>fgoMainWindow</receiver>
   <slot>runSynthesis()</slot>
  </connection>
  <connection>
   <sender>MENU_SCRIPT_GACHAHISTORY</sender>
   <signal>triggered()</signal>
   <receiver>fgoMainWindow</receiver>
   <slot>runGachaHistory()</slot>
  </connection>
  <connection>
   <sender>MENU_SCRIPT_EXPBALL</sender>
   <signal>triggered()</signal>
   <receiver>fgoMainWindow</receiver>
   <slot>expBall()</slot>
  </connection>
  <connection>
   <sender>MENU_CONTROL_MAPKEY</sender>
   <signal>triggered(bool)</signal>
   <receiver>fgoMainWindow</receiver>
   <slot>mapKey(bool)</slot>
  </connection>
  <connection>
   <sender>BTN_STOPLATER</sender>
   <signal>clicked(bool)</signal>
   <receiver>fgoMainWindow</receiver>
   <slot>stopLater(bool)</slot>
  </connection>
  <connection>
   <sender>MENU_SCRIPT_LOTTERY</sender>
   <signal>triggered()</signal>
   <receiver>fgoMainWindow</receiver>
   <slot>runLottery()</slot>
  </connection>
  <connection>
   <sender>MENU_SCRIPT_MAILFILTER</sender>
   <signal>triggered()</signal>
   <receiver>fgoMainWindow</receiver>
   <slot>runMail()</slot>
  </connection>
  <connection>
   <sender>MENU_CONTROL_BENCH</sender>
   <signal>triggered()</signal>
   <receiver>fgoMainWindow</receiver>
   <slot>bench()</slot>
  </connection>
  <connection>
   <sender>MENU_CONTROL_EXEC</sender>
   <signal>triggered()</signal>
   <receiver>fgoMainWindow</receiver>
   <slot>exec()</slot>
  </connection>
  <connection>
   <sender>MENU_SETTINGS_SPECIALDROP</sender>
   <signal>triggered()</signal>
   <receiver>fgoMainWindow</receiver>
   <slot>stopOnSpecialDrop()</slot>
  </connection>
  <connection>
   <sender>MENU_ABOUT_LICENSE</sender>
   <signal>triggered()</signal>
   <receiver>fgoMainWindow</receiver>
   <slot>license()</slot>
  </connection>
  <connection>
   <sender>MENU_CONTROL_169_INVOKE</sender>
   <signal>triggered()</signal>
   <receiver>fgoMainWindow</receiver>
   <slot>invoke169()</slot>
  </connection>
  <connection>
   <sender>MENU_CONTROL_169_REVOKE</sender>
   <signal>triggered()</signal>
   <receiver>fgoMainWindow</receiver>
   <slot>revoke169()</slot>
  </connection>
 </connections>
 <slots>
  <slot>about()</slot>
  <slot>bench()</slot>
  <slot>screenshot()</slot>
  <slot>exec()</slot>
  <slot>explorerHere()</slot>
  <slot>connect()</slot>
  <slot>invoke169()</slot>
  <slot>license()</slot>
  <slot>mapKey(bool)</slot>
  <slot>pause(bool)</slot>
  <slot>revoke169()</slot>
  <slot>runClassic()</slot>
  <slot>runGacha()</slot>
  <slot>runLottery()</slot>
  <slot>runMail()</slot>
  <slot>runMain()</slot>
  <slot>runSynthesis()</slot>
  <slot>runGachaHistory()</slot>
  <slot>expBall()</slot>
  <slot>stop()</slot>
  <slot>stopLater(bool)</slot>
  <slot>stopOnSpecialDrop(bool)</slot>
 </slots>
</ui>
